cli/cmd/sidecar/sidecar.go

```diff
@@ -43,7 +43,7 @@ var (
 	serverType = flag.String("server_type", "sidecar", "The server type to match on health checks")
 
 	listenAddr  = flag.String("listen_addr", "localhost:1991", "Local address to listen on.")
-	besBackend  = flag.String("bes_backend", "", "Server address to proxy build events to.")
+	besBackend  = flag.String("bes_backend", "", "Server address to proxy build events to. Multiple servers could be set using comma ',' separator")
```

Comment:
Multiple servers can be specified if comma-separated.

Reply:
fixed